lib/audit.js

```diff
@@ -25,6 +31,63 @@ auditCmd.completion = function (opts, cb) {
   }
 }
 
+class Auditor extends Installer {
+  constructor (where, dryrun, args, opts) {
+    super(where, dryrun, args, opts)
+    this._deepArgs = (opts && opts.deepArgs) || []
```

Comment:
If this ability is folded into the main installer it shouldn't be underscore prefixed. (Doesn't matter much in a subclass, but I haven't been using that convention elsewhere.)

Reply:
I agree!